lib/logstash/outputs/sns.rb

```diff
@@ -25,7 +25,7 @@
 #  `MAX_MESSAGE_SIZE_IN_BYTES`.
 #
 class LogStash::Outputs::Sns < LogStash::Outputs::Base
-  include LogStash::PluginMixins::AwsConfig
+  include LogStash::PluginMixins::AwsConfig::V2
 
   MAX_SUBJECT_SIZE_IN_CHARACTERS  = 100
   MAX_MESSAGE_SIZE_IN_BYTES       = 32768
```

Comment:
Since we have changed the truncate logic it is a good time to update the documentation for the `format` option and mention when the truncate occur and we do it.